docs/reference/content/getting-started/quick-tour.md

```diff
@@ -350,3 +395,7 @@ collection.bulkWrite(
 {{% note class="important" %}}
 Use of the bulkWrite methods is not recommended when connected to pre-2.6 MongoDB servers, as this was the first server version to support bulk write commands for insert, update, and delete in a way that allows the driver to implement the correct semantics for BulkWriteResult and BulkWriteException. The methods will still work for pre-2.6 servers, but performance will suffer, as each write operation has to be executed one at a time.
 {{% /note %}}
+
+The concludes the quick tour of the new driver. To learn about MongoDB
```

Comment:
I still don't think this in necessary, since right below this link is the Next link to the same page.  And we don't say anything like this on any other page.